espresso-server/app/src/androidTest/java/io/appium/espressoserver/lib/handlers/MoveTo.java

```diff
@@ -0,0 +1,34 @@
+package io.appium.espressoserver.lib.handlers;
+
+import android.support.test.espresso.PerformException;
+import android.support.test.espresso.ViewInteraction;
+import android.view.View;
+
+import javax.annotation.Nullable;
+
+import io.appium.espressoserver.lib.handlers.exceptions.AppiumException;
+import io.appium.espressoserver.lib.model.Element;
+import io.appium.espressoserver.lib.model.MoveToParams;
+import io.appium.espressoserver.lib.viewaction.ScrollTo;
+import io.appium.espressoserver.lib.viewaction.ViewFinder;
+
+public class MoveTo implements RequestHandler<MoveToParams, Void> {
+
+    @Override
+    @Nullable
```

Comment:
Nullable annotation here is redundant. **Void** type cannot have anything else other than null inside %)